bin/install-core

```diff
@@ -37,7 +37,11 @@ export DOTFILES=$SOURCE/dotfiles
 export LOCAL_DIR=$DOTFILES/.local
 
 DOWNLOAD_DIR=$HOME/.local/share/bitcoin-core
-
+BITCOIN_CORE_SOURCE_DOMAIN=(
+    "6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion" #tor
+    "bitcoincore.org" #clearnet
+)
+SECURITY_IN_A_BOX_TOR_DOMAIN="lxjacvxrozjlxd7pqced7dyefnbityrwqjosuuaqponlg3v7esifrzad.onion"
```

Comment:
now this is getting a little crazy, how about hardcode the entire URL here since we're not using two domains to reach the same webpage.

Reply:
Fixed with 24aae9a816aa2e2b34a97278ce3b985f05b71937.